### PR TITLE
[EPROD-560] Use stable storage 0.6

### PR DESCRIPTION
**Pull request by ufoscout**

### CODE REVIEW

**Review by veeso — APPROVED** [review]

File: ic-stable-structures/src/lib.rs [hunk 1]
```diff
@@ -24,3 +24,92 @@ pub use memory_utils::{
 };
 
 pub use structure::*;
+
+// pub mod fff {
```
Comment (GraDKh) [inline ic-stable-structures/src/lib.rs:28]: Commented code

File: ic-stable-structures/src/structure/common/ring_buffer.rs [hunk 2]
```diff
@@ -54,9 +54,15 @@ impl StableRingBufferIndices {
     }
 }
 
+const STABLE_RING_BUFFER_INDICES_SIZE: usize = 2 * size_of::<u64>();
```
Comment (GraDKh) [inline ic-stable-structures/src/structure/common/ring_buffer.rs:57]: Please add newline

**Review by GraDKh — APPROVED** [review]

File: ic-stable-structures/src/structure/common/ring_buffer.rs (hunk 2 above)
Comment (ufoscout) [reply ic-stable-structures/src/structure/common/ring_buffer.rs:57]: done

File: ic-stable-structures/src/lib.rs (hunk 1 above)
Comment (ufoscout) [reply ic-stable-structures/src/lib.rs:28]: removed

**Review by itsyaasir — APPROVED** [review]

**Review by F3kilo — APPROVED** [review]

**Pull request merged**